diff -up ogre-1.12.9/Components/Overlay/CMakeLists.txt.3~ ogre-1.12.9/Components/Overlay/CMakeLists.txt
--- ogre-1.12.9/Components/Overlay/CMakeLists.txt.3~	2020-09-20 04:11:50.000000000 +0200
+++ ogre-1.12.9/Components/Overlay/CMakeLists.txt	2020-11-06 21:33:33.566227453 +0100
@@ -19,14 +19,14 @@ list(APPEND HEADER_FILES
 file(GLOB SOURCE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp")
 
 if(OGRE_BUILD_COMPONENT_OVERLAY_IMGUI)
-  set(IMGUI_DIR "${PROJECT_BINARY_DIR}/imgui-1.77" CACHE PATH "")
+  set(IMGUI_DIR "${PROJECT_BINARY_DIR}/imgui-1.79" CACHE PATH "")
   if(NOT EXISTS ${IMGUI_DIR})
     message(STATUS "Dowloading imgui")
     file(DOWNLOAD
-        https://github.com/ocornut/imgui/archive/v1.77.tar.gz
+        https://github.com/ocornut/imgui/archive/v1.79.tar.gz
         ${PROJECT_BINARY_DIR}/imgui.tar.gz)
     execute_process(COMMAND ${CMAKE_COMMAND}
-        -E tar xf imgui.tar.gz WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
+        -E tar xf imgui.tar.gz WORKING_DIRECTORY ${PROJECT_SOURCE_DIR})
   endif()
   list(APPEND SOURCE_FILES
     ${IMGUI_DIR}/imgui.cpp
@@ -80,4 +80,4 @@ install(FILES ${HEADER_FILES}
 )
 install(FILES "${PROJECT_SOURCE_DIR}/Media/packs/profiler.zip"
   DESTINATION "${OGRE_MEDIA_PATH}/packs/"
-)
\ No newline at end of file
+)
